src/core/PlayAreaApi.ttslua

```diff
@@ -0,0 +1,50 @@
+do
+  local PlayAreaApi = { }
+
+  local PLAY_AREA_GUID = "721ba2"
+
+  -- Returns the current value of the investigator counter from the playmat
+  ---@return Integer.  Number of investigators currently set on the counter
+  PlayAreaApi.getInvestigatorCount = function()
+    return getObjectFromGUID(PLAY_AREA_GUID).call("getInvestigatorCount")
+  end
+
+  -- Move all contents on the play area (cards, tokens, etc) one row up.  Certain fixed objects will
+  -- be ignored, as will anything the player has tagged with 'displacement_excluded'
+  ---@param playerColor Color of the player requesting the shift.  Used solely to send an error
```

Comment:
What do you think about removing the playercolor as argument and instead using "broadcastToAll" for the error message?

Reply:
I can see arguments for either, but I'd opt for less spammy.  In this case, if the script zone isn't found it's basically unrecoverable.  Broadcasting the message to more people who can't do anything about it doesn't seem beneficial.